### PR TITLE
v1.4.0

### LINKED ISSUE #33: Automate release management
Use [GitHub Releases Uploading](https://docs.travis-ci.com/user/deployment/releases/#stq=&stp=0) to automate release management.

### LINKED ISSUE #34: Add code coverage badge
Add https://codecov.io/ badge

### LINKED ISSUE #43: Remove dummy properties
As the bug, that a product must at least have one property is fixed, it is no longer required to export those.

### PR DESCRIPTION
**Pull request by TheKeymaster**
Fixes #34 
Fixes #43 
Fixes #33

### CODE REVIEW

File: _test/template.response.test.php [hunk 1]
```diff
@@ -314,6 +321,17 @@ public function test_response_has_notify_message_when_five_hundred_pages_have_no
...
 
+    public function test_plugin_is_sorted_at_the_top() {
+        $adminPlugin = new admin_plugin_findologicxmlexport();
+        $expectedSortValue = 1;
+        $this->assertEquals($expectedSortValue, $adminPlugin->getMenuSort());
+    }
+
+    public function test_plugin_not_requires_to_be_superuser() {
+        $adminPlugin = new admin_plugin_findologicxmlexport();
+        $this->assertTrue($adminPlugin->forAdminOnly());
```
Comment (georgms) [inline _test/template.response.test.php:332]: The test method says `not requires to be superuser`, yet it checks that it is for `adminOnly`. Is admin < superuser or is the test wrong?

**Review by georgms — CHANGES_REQUESTED** [review]
One small confusion, otherwise mighty fine!

File: _test/template.response.test.php (hunk 1 above)
Comment (TheKeymaster) [reply _test/template.response.test.php:332]: DokuWiki does not display this very good. I think the method name is kinda misleading. As I explained in the DocBlock for the method of the Plugin:

```php
/**
* * **true** => You do not need to be superuser to access this plugin.
* * **false** => You do need to be superuser to access this plugin.
*
* @return bool See method description.
*
*/
```

The method should be renamed to `accessForAdmins()`. As it is a DokuWiki function, renaming is not possible.

Comment (georgms) [reply _test/template.response.test.php:332]: Worst method name ever. 😒

**Review by georgms — APPROVED** [review]

**Pull request merged**